README.md

```diff
@@ -145,8 +145,8 @@ update existing keys in the database.
 Each "op" above corresponds to a write of a single key/value pair.
 I.e., a random write benchmark goes at approximately 400,000 writes per second.
 
-Each "fillsync" operation costs much less (0.3 millisecond)
-than a disk seek (typically 10 milliseconds).  We suspect that this is
+Each "file sync" operation costs much less (0.3 milliseconds)
```

Comment:
"fillsync" is one of our benchmark names, so replacing with "file sync" is incorrect, but thx for "pluralizing" milliseconds.